app/models/ability.rb

```diff
@@ -13,6 +13,10 @@ def initialize(user)
         can :manage, :all
       end
 
+      if (user.submitter? && @user.role == "basic")
```

Comment:
I think a brief comment such as `promote submitters that don't already have higher roles` might help us realize in the future _why_ we are doing this. (non-blocking)